packages/liveblocks-node/src/index.ts

```diff
@@ -7,7 +7,7 @@ detectDupes(PKG_NAME, PKG_VERSION, PKG_FORMAT);
 export type {
   LiveblocksOptions,
   RoomAccesses,
-  RoomInfo,
+  RoomData,
```

Comment:
Should we add this to the end of `src/index.ts` here?

```ts
/**
 * @deprecated RoomInfo was renamed to RoomData, to avoid
 * confusion with the globally augmentable RoomInfo type. This
 * alias will be removed in a future version.
 */
export type RoomInfo = RoomData;
```

This may ease the transition if people upgrade to 2.0. We could remove the alias in 2.1.

Reply:
Good idea yeah, will do 👍